curvefs/src/client/fuse_client.cpp

```diff
@@ -122,17 +122,241 @@ CURVEFS_ERROR FuseClient::Init(const FuseClientOption &option) {
     if (ret3 != CURVEFS_ERROR::OK) {
         return ret3;
     }
-
     ret3 =
         dentryManager_->Init(option.dCacheLruSize, option.enableDCacheMetrics);
     if (ret3 != CURVEFS_ERROR::OK) {
         return ret3;
     }
-
+    warmUpFile_.exist = false;
+    bgCmdStop_.store(false, std::memory_order_release);
+    bgCmdTaskThread_ = Thread(&FuseClient::BackGroundCmdTask, this);
+    taskFetchMetaPool_.Start(WARMUP_THREADS);
     return ret3;
 }
 
+void FuseClient::BackGroundCmdTask() {
+    while (!mounted_.load(std::memory_order_acquire)) {
```

Comment:
Maybe we can start the warmup thread after mount to avoid this sleep？

Reply:
i try it，but the unittest will  have a lot of errors，Since time is tight, and this question is not important. so i have added the todo